waiter/src/waiter/scheduler.clj

```diff
@@ -381,7 +391,7 @@
                                                            (-> instance
```

Comment:
Should we add the status for healthy instances too? `(assoc instance :healthy? true  :health-check-status status)`